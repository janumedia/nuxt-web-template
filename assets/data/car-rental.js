module.exports = {
    carsList:[
        {
            name: "Toyota Avanza",
            img: "images/contents/car-toyota-avanza.jpg",
            desc: "Price With Driver:<br><b>USD 90 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 5 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Suzuki APV",
            img: "images/contents/car-suzuki-apv.jpg",
            desc: "Price With Driver:<br><b>USD 85 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 5 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Suzuki Ertiga",
            img: "images/contents/car-suzuki-ertiga.jpg",
            desc: "Price With Driver:<br><b>USD 90 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 5 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Toyota Innova",
            img: "images/contents/car-toyota-innova.jpg",
            desc: "Price With Driver:<br><b>USD 130 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 5 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Kia Pregio",
            img: "images/contents/car-kia-pregio.jpg",
            desc: "Price With Driver:<br><b>USD 100 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 12 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Kia Travello",
            img: "images/contents/car-kia-travello.jpg",
            desc: "Price With Driver:<br><b>USD 100 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 12 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Isuzu Elf",
            img: "images/contents/car-isuzu-elf.jpg",
            desc: "Price With Driver:<br><b>USD 130 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 12 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Isuzu Elf Long",
            img: "images/contents/car-isuzu-elf-long.jpg",
            desc: "Price With Driver:<br><b>USD 130 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 12 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        },
        {
            name: "Toyota Hi Ace",
            img: "images/contents/car-toyota-hi-ace.jpg",
            desc: "Price With Driver:<br><b>USD 130 / 10 hours</b><br><br>Price Include:<br>Petrol<br>Air condition vehicles<br>Driver speak English<br>Capacity: passenger max 9 pax<br>Departure on Request",
            btnLabel: "Book Now",
            btnUrl: "book-now?type=booking-car"
        }
    ]
}