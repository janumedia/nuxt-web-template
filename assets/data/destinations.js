module.exports = {
    destinationsList:[
        {
            name: "Angel Billabong",
            img: "images/contents/angel-billabong-mid-sq.jpg",
            desc: "a spectacular rock formation on Nusa Penida island's southwestern cliff edges. It's nearby Pasih Uug (Broken Beach), another popular site with unique limestone formations. The naturally formed rock lagoon offers a scenic seascape. You can descend into its crystal-clear rock pools for a swim or a soak",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/angel-billabong",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/angel-billabong.jpg"},
                        {img: "images/contents/angel-billabong-2.jpg"},
                        {img: "images/contents/angel-billabong-3.jpg"},
                        {img: "images/contents/angel-billabong-4.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.733426,
                        long: 115.4470798
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJiZ8Mtmpu0i0R_bG0qk8i-tk'
                    }
                }
            ]
        },
        {
            name: "Broken Beach",
            img: "images/contents/broken-beach-mid-sq.jpg",
            desc: "Broken Beach (Pasih Uug) is a scenic coastal formation on the southwestern edge of Nusa Penida island. The spot is marked by a hilly arch-like rock formation, which is the distinguishable landmark of the area. Set over the crashing waves of the open Indian Ocean, Broken Beach is a great spot for travel photographers and panoramic view seekers. The adjacent area is also home to grey long-tailed macaques. If you’re lucky, you may spot silhouettes of giant mantas near the surface of the deep blue water.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/broken-beach",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/broken-beach.jpg"},
                        {img: "images/contents/broken-beach-2.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7324634,
                        long: 115.449011
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJdX5lGBVu0i0RQ7h9o4ZPdlU'
                    }
                }
            ]
        },
        {
            name: "Kelingking Beach",
            img: "images/contents/klingking-beach-mid-sq.jpg",
            desc: "Kelingking Beach is a hidden beach in the village of Bunga Mekar, on the southwestern coast of Nusa Penida island. You can enjoy one of the most breath-taking views over the hills and small strip of white sand from atop a hill of the same name. The sight features a limestone headland covered in green, against the deep blue waters of the open Indian Ocean. This unique formation is reminiscent of a Tyrannosaurus Rex head, hence its nickname ‘T-Rex Bay’.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/kelingking-beach",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/klingking-beach.jpg", credit: "@journeyera"},
                        {img: "images/contents/klingking-beach-2.jpg", credit: "@mihtiander"},
                        {img: "images/contents/klingking-beach-3.jpg", credit: "@nicmorley"},
                        //{img: "images/contents/tree-of-love.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7527242,
                        long: 115.4697092
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJPeFviSBH0i0R0h1hnHjxmFw'
                    }
                }
            ]
        },
        {
            name: "Peguyangan Waterfall",
            img: "images/contents/peguyangan-waterfall-sq.jpg",
            desc: "Locally referred as Mata Air Peguyangan, is better described as Peguyangan Water Of Springs.<br>With it being more of a religious shrine than a waterfall, many people are disappointed by the end result. But the views on the walk down via the famous Nusa Penida blue stairs more than make up for it.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/peguyangan-waterfall",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/peguyangan-waterfall-1.jpg", credit: "@alexjgillett"},
                        {img: "images/contents/peguyangan-waterfall-2.jpg", credit: "@lolapantravels"},
                        {img: "images/contents/peguyangan-waterfall-3.jpg", credit: "@wanderersandwarriors"},
                        {img: "images/contents/peguyangan-waterfall-4.jpg", credit: "@wanderersandwarriors"},
                        {img: "images/contents/peguyangan-waterfall-5.jpg", credit: "@thenusapenida"},
                        {img: "images/contents/peguyangan-waterfall-6.jpg", credit: "@wanderersandwarriors"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7812546,
                        long: 115.5173131
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJVS_QJn960i0R3Pt_ZCHo0Eo'
                    }
                }
            ]
        },
        {
            name: "Teletubbies Hills",
            img: "images/contents/teletubbies-hills-mid-sq.jpg",
            desc: "Nusa Penida and Bali is identical to the overlay shores, beaches, white sands and sun, but there's something different when you're visiting the island of Nusa Penida. The southeast part of Bali island turn into a nice place to visit and you can find a similar green hills like in a movie called Teletubbies Hill",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/teletubbies-hills",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/teletubbies-hills.jpg"},
                        {img: "images/contents/teletubbies-hills-2.jpg", credit: "@galuh.pj"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7853848,
                        long: 115.5725955
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJYfEnJEV30i0RS9pcaNa604M'
                    }
                }
            ]
        },
        {
            name: "Raja Lima",
            img: "images/contents/raja-lima-mid-sq.jpg",
            desc: "The exquisite views of the beach area are undoubtedly. Beach, cliffs and Indonesian Ocean seem obvious. Small island is similar to Raja Ampat, West Papua. Many tourists call Atuh Beach areas with the nickname of Thousand Islands. This appeal become the choice of tourists visiting Nusa Penida. Cliffs and hills that surround the coast as shoulder to shoulder shows the beauty of the Atuh Beach overall.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/raja-lima",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/raja-lima.jpg"},
                        {img: "images/contents/tree-house.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7787791,
                        long: 115.614736
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJ419QjYF30i0R_Bn1jT5cVlo'
                    }
                }
            ]
        },
        {
            name: "Atuh Beach",
            img: "images/contents/atuh-beach-mid-sq.jpg",
            desc: "You will be amazed by the clarity of the turquoise waters, the gorgeous white sandy beaches, the towering high cliff walls, beautiful green nature, and best of all it should be all yours, your own private beach!",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/atuh-beach",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/atuh-beach.jpg"},
                        {img: "images/contents/atuh-beach-2.jpg"},
                        {img: "images/contents/atuh-beach-3.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7734116,
                        long: 115.61963
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJhYE8QJt30i0Rs8qDtRiE1iU'
                    }
                }
            ]
        },
        {
            name: "Crystal Bay",
            img: "images/contents/crystal-bay-beach-sq.jpg",
            desc: "Located on the west of Nusa Penida, is a suitable place to enjoy the beautiful views of the bay and rocky island. There you can sunbathe, swim and snorkel from the shore. Crystal Bay is very famous for Mola - mola (sunfish), so do not be surprised if you see a lot of dive boats anchored here.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/crystal-bay-beach",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/crystal-bay-beach-2.jpg"},
                        {img: "images/contents/crystal-bay-beach.jpg"},
                        {img: "images/contents/snorkeling-crystal-bay-beach.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7155815,
                        long: 115.4569574
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJrZrXTwhu0i0R1-hceK52v_o'
                    }
                }
            ]
        },
        {
            name: "Devil's Tears",
            img: "images/contents/devils-tears-mid-sq.jpg",
            desc: "Devil's Tears is a sparsely vegetated outcrop of land with dramatic limestone cliffs, on the west coast of Nusa Lembongan. Devil's Tears is situated right next to the stunning Dream Beach, where turtles play in the waves and lay their eggs on the beach. This is a magnificent spot to visit. Devil's Tears provides an area with many viewing spots for watching the massive swell crash against the cliffs below and spray high into the air.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/devils-tears",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/devils-tears.jpg"},
                        {img: "images/contents/devils-tears-2.jpg"},
                        {img: "images/contents/devils-tears-3.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.690743,
                        long: 115.4280638
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJNXQYfLRt0i0RALc3MIMT6QA'
                    }
                }
            ]
        },
        {
            name: "Giri Putri Cave",
            img: "images/contents/giri-putri-cave-mid-sq.jpg",
            desc: "Giri Putri cave is located near the village of Karangsari. After climbing a hundred works, you will reach the entrance of the cave so narrow that must kneel to enter. Inside, we are surprised by its volume. It is very dark inside, but at the end of the cave there is a large opening, and we do not know if the back wall was opened intentionally or if it was like that originally. Close to the exit, you will discover a place of worship for Hindu devotees, but also for the Buddhist and Confucian followers, evidenced by the presence of a statue of Dewi Kwan Im.",
            btnLabel: "Make a Tour",
            btnUrl: "/tours",
            detailUrl: "/destinations/giri-putri-cave",
            details:[
                {
                    name: "Gallery",
                    slides: [
                        {img: "images/contents/giri-putri-cave-2.jpg"},
                        {img: "images/contents/giri-putri-cave-3.jpg"},
                        {img: "images/contents/giri-putri-cave.jpg"}
                    ]
                },
                {
                    googleMap: {
                        lat: -8.7089515,
                        long: 115.583926
                    }
                },
                {
                    name: "Reviews",
                    googleReview: {
                        placeID: 'ChIJf6i9TaF20i0R8_IaKI-kqMQ'
                    }
                }
            ]
        }
    ]
}