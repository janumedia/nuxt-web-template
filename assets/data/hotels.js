module.exports = {
    hotelsList:[
        {
            name: "Nusa Penida Beach Hotel",
            img: "images/contents/hotel-nusa-penida-beach-mid.jpg",
            desc: "Conveniently located in Bali, Nusa Penida Beach Hotel is a great base from which to explore this vibrant city. Situated only 20 km from the city center, guests are well located to enjoy the town's attractions and activities. With its convenient location, the hotel offers easy access to the city's must-see destinations.<br><br>Nusa Penida Beach Hotel also offers many facilities to enrich your stay in Bali. This hotel offers numerous on-site facilities to satisfy even the most discerning guest.",
            btnLabel: "Book Now",
            btnUrl: "/book-now?type=booking-tour"
        },
        {
            name: "Bintang Bungalows",
            img: "images/contents/hotel-bintang-bungalows-mid.jpg",
            desc: "Bintang Bungalow is located in Toyapakeh just about 800 meters from Crystal Bay. The hotel is a 15-minutes drive to Gamat Bay and 30-minutes to Angel's Billabong.<br><br>Features an outdoor pool. You can enjoy meals at the restaurant at the accommodation.",
            btnLabel: "Book Now",
            btnUrl: "/book-now?type=booking-tour"
        },
        {
            name: "Caspla Beach Hotel",
            img: "images/contents/hotel-caspla-beach-mid.jpg",
            desc: "Caspla Beach Hotel, Restaurant & Bar is built seashore side on the easthern most tip of Bali, that is Nusa Penida Island.<br><br>Enjoy luxury hotels on east Bali and an unforgettable holiday in Caspla Beach Hotel, Bar & Restaurant unique, elegant, tropical style, ocean front bungalows. the hotel building full of wood, here you can enjoy free pool, eating at the beachside restaurant.",
            btnLabel: "Book Now",
            btnUrl: "/book-now?type=booking-tour"
        },
        {
            name: "Semabu Hills Hotel",
            img: "images/contents/hotel-semabu-hills-mid.jpg",
            desc: "It can be argued that Bali is as close to heaven on earth as it gets and we at Semabu Hills have put in place the crowning glory of this picturesque getaway.<br><br>Just 30 minutes off the east coast of Bali on the tropical island of Nusa Penida, Semabu Hills is proud to host you and give you an experience to cherish.",
            btnLabel: "Book Now",
            btnUrl: "/book-now?type=booking-tour"
        }
    ]
}