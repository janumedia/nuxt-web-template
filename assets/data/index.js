module.exports = {
    destinationList: [
        {
            id:0,
            name: "One Day Tour Nusa Penida",
            img: "images/contents/crystal-bay-beach-mid-sq.jpg",
            desc: "One day tour exploring Nusa Penida:<br>Tree of Love and Klingking Beach,<br>Broken Beach and Angel Billabong,<br>Crystal Bay Beach<br><br><b>Start from Rp 400.000</b>",
            btnLabel: "More Details",
            url: "/tours/one-day-tour-nusa-penida"
        },
        {
            id: 1,
            name: "Snorkeling at Nusa Penida Beach",
            img: "images/contents/snorkeling-crystal-bay-beach-mid-sq.jpg",
            desc: "Enjoy beautiful underwater scene in Nusa Penida Beach<br>If you're lucky you may met Manta Ray or Mola-mola<br><br><b>Start from Rp 600.000</b>",
            btnLabel: "More Details",
            url: "/tours/one-day-tour-nusa-penida-plus-snorkeling"
        },
        {
            id: 2,
            name: "2 Days and 1 Night Tour Nusa Penida",
            img: "images/contents/raja-lima-sunset-mid-sq.jpg",
            desc: "Complete 2 days and 1 night tour covering west and east side of Nusa Penida famous tourist destionations<br><br><b>Start from Rp 750.000</b>",
            btnLabel: "more details",
            url: "/tours/2-days-1-night-tour-nusa-penida"
        },
        {
            id: 3,
            name: "One Day Tour Nusa Lembongan",
            desc: "One day tour within Nusa Lembongan:<br>Panorama Point, Dream Beach, Devil's Tears, Yellow Bridge and Mangrove Forest<br><br><b>Start from Rp 450.000</b>",
            img: "images/contents/devils-tears-mid-sq.jpg",
            btnLabel: "more details",
            url: "/tours/one-day-tour-nusa-lembongan"
        },
        {
            id: 4,
            name: "Nusa Penida Magical Place",
            desc: "You will guided to visiting Giri Putri Cave, a magical place inside the cave where you will discover a place of worship Hindu devotees and also Buddist and Confucian followers<br><br><b>Start from Rp 750.000</b>",
            img: "images/contents/giri-putri-cave-mid-sq.jpg",
            btnLabel: "more details",
            url: "/tours/2-days-1-night-tour-nusa-penida"
        }
        
    ]
}