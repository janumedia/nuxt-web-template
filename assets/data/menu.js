module.exports = {
    menu: [
        {
            name: "Home", path: "/", quote: "Cherysa Tour is a Bali and Nusa Penida Tour Agency"
        },
        {
            name: "Destinations", path: "/destinations", submenu: [
                { name: "Angel Billabong", path: "/destinations/angel-billabong"},
                { name: "Broken Beach", path: "/destinations/broken-beach"},
                { name: "Kelingking Beach", path: "/destinations/kelingking-beach"}
            ]
        },
        {
            name: "Hotels", path: "/hotels", quote: "Hotels in Nusa Penida Partnership with Cherysa Tour"
        },
        {
            name: "Contact", path: "/contact"
        },
        {
            name: "Sitemap", path: "/sitemap", mobileOnly: true
        }
    ]
}