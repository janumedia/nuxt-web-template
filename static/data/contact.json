{
    "title": "Contact Us",
    "desc": "Information & Reservation Board",
    "metaImage": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fklingking-beach-2.jpg?alt=media&token=f7b4c2d0-39c5-482e-a929-5a696aa5be18"
}