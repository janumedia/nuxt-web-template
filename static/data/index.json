{
    "title": "Nusa Penida Tour Guide",
    "desc": "Totalling some 200 square kilometres, Nusa Penida is much larger than the better known Nusa Lembongan. However, tourist infrastructure is very limited here but growing fast.\n\nDue to a lack of natural fresh water, little is grown or produced on Nusa Penida, and even some basic foodstuffs come in by boat. Visitors should therefore expect higher prices than in Bali, and not bank on any tourism-related luxury items being available for purchase here.",
    "metaImage": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fcrystal-bay-beach-2.jpg?alt=media&token=be8ffb8f-17e9-4b85-bf4a-9dff7f38348a",
    "body": [
        {
            "listBox": [
                {
                    "slides": [
                        {"name": "Peguyangan Water Spring", "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fpeguyangan-waterfall-3.jpg?alt=media&token=82e2b880-f354-468e-a857-f341a108e239", "credit": "@wanderersandwarriors"},
                        {"name": "Kelingking Beach", "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fklingking-beach.jpg?alt=media&token=0608592e-0bca-43ce-b535-857065b67ef5"},
                        {"name": "Angel Billabong", "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fangel-billabong.jpg?alt=media&token=9d4b2f99-3667-464e-9d6a-5c4b6850bf84"},
                        {"name": "Broken Beach", "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fbroken-beach.jpg?alt=media&token=d5c744e6-9fd4-4df0-8552-03659675f6d7"},
                        {"name": "Crystal Bay Beach", "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/slides%2Fcrystal-bay-beach-2.jpg?alt=media&token=be8ffb8f-17e9-4b85-bf4a-9dff7f38348a"}
                    ]
                }
            ]
        },
        {
            "name": "Recommendations",
            "chessGrid": [
                {
                    "name": "Angel Billabong",
                    "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/angel-billabong-mid-sq.jpg?alt=media&token=ef70e878-0de2-4b0c-bed8-513d3bc21758",
                    "desc": "a spectacular rock formation on Nusa Penida island's southwestern cliff edges. It's nearby Pasih Uug (Broken Beach), another popular site with unique limestone formations. The naturally formed rock lagoon offers a scenic seascape. You can descend into its crystal-clear rock pools for a swim or a soak",
                    "btnLabel": "View details",
                    "url": "/destinations/angel-billabong"
                },
                {
                    "name": "Broken Beach",
                    "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/broken-beach-mid-sq.jpg?alt=media&token=9769ac16-f00a-494d-afb1-709e96327d6e",
                    "desc": "Broken Beach also popular as Pasih Uug is a scenic coastal formation on the southwestern edge of Nusa Penida island. The spot is marked by a hilly arch-like rock formation, which is the distinguishable landmark of the area. Set over the crashing waves of the open Indian Ocean.",
                    "btnLabel": "View details",
                    "url": "/destinations/broken-beach"
                },
                {
                    "name": "Kelingking Beach",
                    "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/klingking-beach-mid-sq.jpg?alt=media&token=fe9953a8-95d2-4c70-97c8-ae8df6816bdb",
                    "desc": "Kelingking Beach is a hidden beach in the village of Bunga Mekar, on the southwestern coast of Nusa Penida island. You can enjoy one of the most breath-taking views over the hills and small strip of white sand from atop a hill of the same name. This unique formation is reminiscent of a Tyrannosaurus Rex head, hence its nickname ‘T-Rex Bay’.",
                    "btnLabel": "View details",
                    "url": "/destinations/kelingking-beach"
                },
                {
                    "name": "Angel Billabong",
                    "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/angel-billabong-mid-sq.jpg?alt=media&token=ef70e878-0de2-4b0c-bed8-513d3bc21758",
                    "desc": "a spectacular rock formation on Nusa Penida island's southwestern cliff edges. It's nearby Pasih Uug (Broken Beach), another popular site with unique limestone formations. The naturally formed rock lagoon offers a scenic seascape. You can descend into its crystal-clear rock pools for a swim or a soak",
                    "btnLabel": "View details",
                    "url": "/destinations/angel-billabong"
                },
                {
                    "name": "Broken Beach",
                    "img": "https://firebasestorage.googleapis.com/v0/b/nuxt-web-template.appspot.com/o/broken-beach-mid-sq.jpg?alt=media&token=9769ac16-f00a-494d-afb1-709e96327d6e",
                    "desc": "Broken Beach also popular as Pasih Uug is a scenic coastal formation on the southwestern edge of Nusa Penida island. The spot is marked by a hilly arch-like rock formation, which is the distinguishable landmark of the area. Set over the crashing waves of the open Indian Ocean.",
                    "btnLabel": "View details",
                    "url": "/destinations/broken-beach"
                }
            ]
        }
    ],
    "extras": [
        {
            "name": "Review",
            "googleReview": {
                "placeID": "ChIJ0xkTTRlx0i0Re3sZsgY3Olw"
            }
        }
    ]
}