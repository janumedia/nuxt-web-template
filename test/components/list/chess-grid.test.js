import { shallowMount } from '@vue/test-utils'
import NuxtLink from '@/.nuxt/components/nuxt-link'
import ChessGrid from '@/components/list/chess-grid'
import data from '../../fakeData/chess-grid-data'

describe("chess-grid.vue", () => {
    
    let wrapper,
        items = data.chessGrid;
    
    beforeEach(() => {
        ChessGrid.components = ChessGrid.components || {}
        ChessGrid.components.NuxtLink = NuxtLink
        wrapper = shallowMount(ChessGrid, {
            propsData: {
                list: items
            }
        })
    });

    it('list is Array', () => {
        expect(wrapper.vm.list.constructor === Array).toBeTruthy();
    });

    it('should render chess-grid init state', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });
})