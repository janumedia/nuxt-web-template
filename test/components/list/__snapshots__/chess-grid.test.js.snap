// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chess-grid.vue should render chess-grid init state 1`] = `
<div class="chess-grid">
  <div class="chess-grid__item">
    <div class="chess-grid__item-box  chess-grid__item-box--pull">
      <linkedimage-stub verticalalign="true" square="true" title="Name A" url="/link/to/a" src="imageA.jpg"></linkedimage-stub>
    </div>
    <div class="chess-grid__item-box chess-grid__item-info  chess-grid__item-box--push">
      <div class="v-align-pos">
        <h3>Name A</h3>
        <p>Description A</p>
        <button-stub url="/link/to/a" label="View details"></button-stub>
      </div>
    </div>
  </div>
  <div class="chess-grid__item">
    <div class="chess-grid__item-box ">
      <linkedimage-stub verticalalign="true" square="true" title="Name B" url="/link/to/b" src="imageB.jpg"></linkedimage-stub>
    </div>
    <div class="chess-grid__item-box chess-grid__item-info ">
      <div class="v-align-pos">
        <h3>Name B</h3>
        <p>Description B</p>
        <button-stub url="/link/to/b" label="View details"></button-stub>
      </div>
    </div>
  </div>
</div>
`;
